version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.0-preview",
+  "version": "1.7-preview",
```

Comment:
Is it the correct version?

Reply:
https://github.com/dbos-inc/ttdbg-extension/pull/44#issuecomment-2699042683